tuf/exceptions.py

```diff
@@ -252,16 +253,17 @@ class InvalidNameError(Error):
 class UnsignedMetadataError(Error):
   """Indicate metadata object with insufficient threshold of signatures."""
 
-  def __init__(self, message, signable):
+  # signable is not used but kept in method signature for backwards compat
```

Comment:
Do we want a FIXME or similar here? So that we are reminded to remove this unused variable before we tag 1.0?